pages/sections/landing.tsx

```diff
@@ -21,7 +21,8 @@ import bird1 from '../../public/assets/bird1.svg';
 import bird2 from '../../public/assets/bird2.svg';
 import bird3 from '../../public/assets/bird3.svg';
 import star from '../../public/assets/star.svg';
-import globeLight from '../../public/assets/globeLight.svg';
+import globeLight from '../../public/assets/globelight.svg';
+import globeDark from '../../public/assets/globedark.svg';
```

Comment:
can we use `@/public/assets/___` from here on out i think its mesin with deployment also cleaner :)))